sort.py

```diff
@@ -0,0 +1,22 @@
+
+def sort(array):
+    if not array or len(array) <= 1:
+        return array # nothing to sort
+
+    index = 0
+    swapped = True
```

Comment:
What is this flag for? It seems a bit redundant and I'm curious about its usage here.